Mathlib/CategoryTheory/Functor/Const.lean

```diff
@@ -99,6 +99,13 @@ def compConstIso (F : C ⥤ D) :
     (fun X => NatIso.ofComponents (fun j => Iso.refl _) (by aesop_cat))
     (by aesop_cat)
 
+/-- The canonical isomorphism
+`const D ⋙ (whiskeringLeft J _ _).obj F ≅ const J`.-/
+@[simps!]
+def constCompWhiskeringLeftIso (F : J ⥤ D) :
```

Comment:
I think there is a dual statement to this, namely

```lean
@[simps!]
def constCompWhiskeringRightIso (F : D ⥤ C) :
    const J ⋙ (whiskeringRight J D C).obj F ≅ F ⋙ const J :=
  NatIso.ofComponents fun X => NatIso.ofComponents fun Y => Iso.refl _
```

Reply:
Yes, I think I'll just add the dual, it's definitely a useful statement

Reply:
The dual is right above, as `compConstIso`

Reply:
Oops, sorry about that.